src/streaming/protection/servers/Primetime.js

```diff
@@ -29,17 +29,32 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-/**
- * Adobe Access DRM
- *
- * @class
- * @implements MediaPlayer.dependencies.protection.KeySystem
- */
-MediaPlayer.dependencies.protection.KeySystem_Access = function() {
+MediaPlayer.dependencies.protection.servers.Primetime = function() {
     "use strict";
-};
 
-MediaPlayer.dependencies.protection.KeySystem_Access.prototype = {
-    constructor: MediaPlayer.dependencies.protection.KeySystem_Access
+    return {
+
+        getServerURLFromMessage: function(url, message, messageType) {
+            if (messageType === "individualization-request") {
+                url = 'http://individualization.adobe.com/flashaccess/i15n/v5';
```

Comment:
right now I do not have enough information to answer this question. We are not working directly with Adobe but via our DRM integration partner. We plan to have a first draft at the end of December.